src/cmd/services/m3dbnode/main/main_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/m3db/m3cluster/shard"
+
```

Comment:
nit: why the extra line between the imports below it.

Reply:
unintentional, fixed